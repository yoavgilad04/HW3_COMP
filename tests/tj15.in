//redef in function

int foo(int bar, 
bool bar,
byte bar){ return 1;}

void main(){
    return;
}