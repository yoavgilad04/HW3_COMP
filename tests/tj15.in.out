line 4: identifier bar is already defined
